server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -157,30 +146,26 @@ public StatusDTO status() {
 
         good = good && (mode == Mode.NORMAL);
 
-        StatusDTO status;
+        StatusDTO status = new StatusDTO()
+            .result(good)
+            .version(version)
+            .release(release)
+            .standalone(standalone)
+            .rulesVersion(jsProvider.getRulesVersion())
+            .rulesSource(rulesSource != null ? rulesSource.toString() : null)
+            .mode(mode != null ? mode.toString() : null)
+            .modeReason(mcr != null ? mcr.toString() : null)
+            .modeChangeTime(mcr != null ? mcr.getTime().toInstant().atOffset(ZoneOffset.UTC) : null)
+            .managerCapabilities(caps.stream().collect(Collectors.toList()))
+            .timeUTC(new Date().toInstant().atOffset(ZoneOffset.UTC));
```

Comment:
There is no need to create a Date instance first and then convert it to OffsetDateTime here. `OffsetDateTime.now()` will do.

Reply:
Done.